Pod/Classes/FBSnapshotTestCase+AGSnapshotHelper.m

```diff
@@ -19,11 +18,11 @@ - (void)testView:(nonnull UIView *)view {
     [self testView:view contentSizeCategory:nil identifier:nil];
 }
 
-- (void)testView:(nonnull UIView *)view contentSizeCategory:(nonnull NSString *)contentSizeCategory {
+- (void)testView:(nonnull UIView *)view contentSizeCategory:(nonnull UIContentSizeCategory)contentSizeCategory {
     [self testView:view contentSizeCategory:contentSizeCategory identifier:nil];
 }
 
-- (void)testView:(nonnull UIView *)view contentSizeCategory:(nonnull NSString *)contentSizeCategory windowsBoundsSizesMask:(AGWindowBoundsSizesMask)mask {
+- (void)testView:(nonnull UIView *)view contentSizeCategory:(nonnull UIContentSizeCategory)contentSizeCategory windowsBoundsSizesMask:(AGWindowBoundsSizesMask)mask {
```

Comment:
IMO `nonnull` it's not necessary, because is already defined in interface file. If you decide to change it there you need to remember to change it also in implementation file, which is prone to error.

Reply:
I know `nonnull` is not a must in implementation file, but IMO information about nullability in implementation file is very useful. Developers spent most of the time in `.m` instead of `.h`, so it's nice to just see nullability annotation instead of analysing code.
In fact in swift files you don't have header files, and nullability is everywhere. I like that ;)

Reply:
If we follow this logic we should also copy the documentation from interface file.

Reply:
This class will be rewritten to Swift anyway 😉